.circleci/config.yml

```diff
@@ -156,7 +156,7 @@ jobs:
           name: ️️🏗️ build misc
           command: |
               . venv/bin/activate && pip install --upgrade -e . --quiet && mkdir packages
-              git clone --depth 1 https://github.com/plotly/dash-table.git
+              git clone --depth 1 -b persistence https://github.com/plotly/dash-table.git
```

Comment:
The test failures look as though I didn't get the right branch of table installed. @byronz did I do something wrong here? I want to use https://github.com/plotly/dash-table/pull/566